docs/ref/class-based-views/mixins-editing.txt

```diff
@@ -51,7 +51,7 @@ The following mixins are used to construct Django's editing views:
     .. method:: get_form_class()
 
         Retrieve the form class to instantiate. By default
-        :attr:`.form_class`.
+        :attr:`~django.views.generic.edit.FormMixin.form_class`.
```

Comment:
This caused a warning:

_"django/docs/ref/class-based-views/mixins-editing.txt:53: WARNING: more than one target found for cross-reference 'form_class': django.views.generic.edit.FormMixin.form_class, django.contrib.auth.views.PasswordChangeView.form_class, django.contrib.auth.views.PasswordResetView.form_class, django.contrib.auth.views.PasswordResetConfirmView.form_class"_.